docs/book/src/roadmap.md

```diff
@@ -2,56 +2,51 @@
 
 This roadmap is a constant work in progress, subject to frequent revision. Dates are approximations.
 
-## Ongoing
 
-- Documentation improvements
+## v0.3.7 (v1alpha3+) ~ June/July 2020
+
+|Area|Description|Issue/Proposal|
+|--|--|--|
+|Testing|E2E Test plan|[Spreadsheet](https://docs.google.com/spreadsheets/d/1uB3DyacOLctRjbI6ov7mVoRb6PnM4ktTABxBygt5sKI/edit#gid=0)
+|Testing|Enable webhooks in integration tests|[#2788](https://github.com/kubernetes-sigs/cluster-api/issues/2788)|
+|Control Plane|KubeadmControlPlane robustness|[#2753](https://github.com/kubernetes-sigs/cluster-api/issues/2753)|
+|Control Plane|KubeadmControlPlane adoption|[#2214](https://github.com/kubernetes-sigs/cluster-api/issues/2214)|
+|Extensibility|Clusterctl library should support extensible templating|[#2339](https://github.com/kubernetes-sigs/cluster-api/issues/2339)|
+|Dependency|Clusterctl manages cert-manager lifecycle|[#2635](https://github.com/kubernetes-sigs/cluster-api/issues/2635)|
```

Comment:
I'm not sure this will get ready for the milestone, AFAIK there is no a design yet